### DISCUSSION TITLE
Circle around WordPress icon

### DISCUSSION THREAD

**Discussion by ShahriarKh** [tabler/tabler-icons#286] [Ideas]
I believe adding a circle around the current `W` in WordPress icon makes it more clear. Almost every version of the WordPress logo I see has that circle, whether the icon is in outline style, duotone, single-color, ...

A rough draft in Paint 😁:  
![image](https://user-images.githubusercontent.com/31452340/184364525-42349391-5c83-4038-a2db-c25dbb61d78c.png)

**Comment by codecalm** [comment]
something about that? 

![Screenshot 2022-08-12 at 16 30 13](https://user-images.githubusercontent.com/1282324/184375934-925f6405-c0b0-485c-956c-2b1bb073c9e4.png)

**Reply by ShahriarKh** [reply]
The "W" in the original logo is intersecting with the circle.
![image](https://user-images.githubusercontent.com/31452340/184388204-74525ad3-4671-4a48-8a5b-2a9982491896.png)
